tasks.py

```diff
@@ -24,14 +24,20 @@
 
 
 @task(
-    iterable=["component"],
+    iterable=["component", "docker_args"],
     help={
         "component": "The components to build - if none given defaults to: "
         + ", ".join(ALL_COMPONENTS),
         "dry_run": "Do not perform any changes, just generate configs and log what would be done",
+        "docker_args": (
+                "Arguments to build docker images. "
+                + "Passed as --build-arg to docker build. If you want to"
```

Comment:
"If you want to build several components, pass them as separate arguments" is very unclear

Reply:
It's not really much clearer still. What are you trying to say? Like that several arguments would be `--build-arg foo=bar --build-arg foo2=bar2`?

Reply:
In that case, maybe `Repeat for multiple build arguments`?

Reply:
Updated